src/Security/Authentication/Twitter/src/TwitterExtensions.cs

```diff
@@ -25,5 +33,26 @@ public static AuthenticationBuilder AddTwitter(this AuthenticationBuilder builde
             builder.Services.TryAddEnumerable(ServiceDescriptor.Singleton<IPostConfigureOptions<TwitterOptions>, TwitterPostConfigureOptions>());
             return builder.AddRemoteScheme<TwitterOptions, TwitterHandler>(authenticationScheme, displayName, configureOptions);
         }
+
+        public static async Task EnsureTwitterRequestSuccess(this HttpResponseMessage response)
```

Comment:
Move it back into TwitterHandler.cs as a normal helper method, it's not used anywhere else.